examples/chip-tool/commands/interactive/WebSocketServer.cpp

```diff
@@ -145,3 +155,15 @@ CHIP_ERROR WebSocketServer::Run(chip::Optional<uint16_t> port, WebSocketServerDe
     lws_context_default_loop_run_destroy(context);
     return CHIP_NO_ERROR;
 }
+
+CHIP_ERROR WebSocketServer::Send(const char * msg)
+{
+    VerifyOrReturnError(nullptr != gWebSocketInstance, CHIP_ERROR_INCORRECT_STATE);
+
+    auto buffer = static_cast<unsigned char *>(malloc(LWS_PRE + strlen(msg)));
```

Comment:
should we test for malloc failure?

Reply:
also since you alloc and free, how about a scoped buffer, like `chip::Platform::ScopedMemoryBuffer`?

Reply:
Fixed.